src/main/java/teammates/logic/automated/FeedbackSessionOpeningMailAction.java

```diff
@@ -59,25 +59,24 @@ protected void doPostProcessingForSuccesfulSend() throws InvalidParametersExcept
     protected List<MimeMessage> prepareMailToBeSent() 
             throws MessagingException, IOException, EntityDoesNotExistException {
         Emails emailManager = new Emails();
-        List<MimeMessage> preparedEmails = null;
         
         FeedbackSessionAttributes feedbackObject = FeedbackSessionsLogic.inst()
                 .getFeedbackSession(feedbackSessionName, courseId);
         log.info("Fetching feedback session object for feedback session name : "
                 + feedbackSessionName + " and course : " + courseId);
         
-        if (feedbackObject != null) {
-             /*
-              * Check if feedback session was deleted between scheduling
-              * and the actual sending of emails
-              */
-            preparedEmails = emailManager
-                            .generateFeedbackSessionOpeningEmails(feedbackObject);
+        if (feedbackObject == null) {
+            log.severe("Feedback session object for feedback session name : " + feedbackSessionName 
+                       + " for course : " + courseId + " could not be fetched");
+            return null;
         } else {
```

Comment:
Similar to what you did in `FeedbackSessionClosingMailAction.java`, maybe you could move the instantiation of `emailManager` to here where it is used, and remove the `else` clause